### DISCUSSION TITLE
k3s version과 kubeflow version이 맞춰진 것에 대해 궁금합니다.

### DISCUSSION THREAD

**Discussion by ChaejinE** [mlops-for-all/mlops-for-all.github.io#112] [Q&A] [answered]
k3s는 1.2.7, kubeflow는 1.4로 알구있는데요. 
제가 mac m1 silicon 이다보니 기존 docs 대로 버전을 설치했을 때, istio-system namespace가 pending 상태입니다. (multipass로 만든 가상환경을 클러스터로 사용 중입니다.)
혹시 reference 하신 버전들이 있을까요 ? 저는 더 높은 버전에서 한번 시도해보려합니다.

**Comment by anencore94** [comment, marked as answer]
Reference 한 버전들이라는 게 정확히 어떤 걸 여쭤보시는 걸까요?
해당 버전을 사용한 이유를 여쭤보시는 것이라면 릴리즈 기준 최신 버전이었기 때문입니다.

다른 버전을 사용하실 예정이라면, kubeflow v1.4 를 사용한다면 k3s 사용 시 k8s version 을 v1.21 이하로 사용하셔야 하고, k8s version v1.22 ~ v1.25 에 대한 지원은 kubeflow v1.6 부터 가능합니다. [ref](https://blog.kubeflow.org/kubeflow-1.6-release/)

namespace pending 인 이유는 상황에 따라 다양할텐데, 더 구체적인 원인 파악이 필요해보입니다. 정확히 namespace 가 pending 인 건지, namespace 내부의 pod 들이 pending 인 것인지 문의드리며, namespace 가 pending 인 경우 namespace describe 시 보다 구체적인 이유가 담겨 있을 것입니다.
